lessons/second.rb

```diff
@@ -0,0 +1,28 @@
+
+people = {}
+
+puts "What is your first name?"
+people[:first_name] = gets.chomp
+puts "What is your last name?"
+people[:last_name] = gets.chomp
+puts "What is your age?"
+people[:age] = gets.chomp
```

Comment:
not required by the assignment but consider a check for numeric input

Reply:
done